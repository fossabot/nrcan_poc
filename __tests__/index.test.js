import { shallow } from 'enzyme'
import React from 'react'

import App from '../pages/index.js'

describe('IndexPage', () => {
  it('shows "Fully-automated luxury deployments!"', () => {
    const app = shallow(<App />)
    expect(app.find('p').text()).toEqual('Fully-automated luxury deployments!')
  })
})
