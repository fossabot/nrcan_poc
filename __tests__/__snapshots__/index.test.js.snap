// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`With Snapshot Testing App shows "Hello world!" 1`] = `
<div
  className="jsx-152973761"
>
  <p
    className="jsx-152973761"
  >
    Hello World!
  </p>
</div>
`;
